pkgs/games/doom-ports/slade/default.nix

```
@@ -71,7 +65,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Doom editor";
     homepage = "http://slade.mancubus.net/";
-    license = licenses.gpl2Plus;
+    license = licenses.gpl2Only;
```

```suggestion
    license = licenses.gpl2Plus;
```
See e.g. https://github.com/sirjuddington/SLADE/blob/master/src/TextEditor/Lexer.cpp#L14

Unless I missed something else in the repository.

---

I made that change because of this commit: https://github.com/sirjuddington/SLADE/commit/d4692100bccc641c874f0398f0771ae19115f443

---

Opened https://github.com/sirjuddington/SLADE/issues/1754